binding/go/pv_recorder.go

```diff
@@ -157,31 +155,42 @@ func (pvrecorder *PvRecorder) Stop() error {
 }
 
 // Read function reads audio frames.
-func (pvrecorder *PvRecorder) Read() ([]int16, error) {
-	pcm := C.malloc_cgo(C.int32_t(pvrecorder.FrameLength))
+func (pvRecorder *PvRecorder) Read() ([]int16, error) {
+	pcm := C.malloc_cgo(C.int32_t(pvRecorder.FrameLength))
 	defer C.free(unsafe.Pointer(pcm))
 
-	ret := nativePvRecorder.nativeRead(pvrecorder, pcm)
+	ret := nativePvRecorder.nativeRead(pvRecorder, pcm)
 	if ret != SUCCESS {
 		return nil, fmt.Errorf("PvRecorder Read failed with: %s", pvRecorderStatusToString(ret))
 	}
 
-	pcmCSlice := (*[1 << 28]C.int16_t)(unsafe.Pointer(pcm))[:pvrecorder.FrameLength:pvrecorder.FrameLength]
-	pcmSlice := make([]int16, pvrecorder.FrameLength)
+	pcmCSlice := (*[1 << 28]C.int16_t)(unsafe.Pointer(pcm))[:pvRecorder.FrameLength:pvRecorder.FrameLength]
+	pcmSlice := make([]int16, pvRecorder.FrameLength)
 	for i := range pcmSlice {
 		pcmSlice[i] = int16(pcmCSlice[i])
 	}
 
 	return pcmSlice, nil
 }
 
+// SetDebugLogging enables or disables debug logging for PvRecorder. Debug logs will indicate when there are
+// overflows in the internal  frame buffer and when an audio source is generating frames of silence.
+func (pvRecorder *PvRecorder) SetDebugLogging(isDebugLoggingEnabled bool) {
+	nativePvRecorder.nativeSetDebugLogging(pvRecorder, isDebugLoggingEnabled)
+}
+
+// GetIsRecording gets whether the given instance is currently recording audio or not.
+func (pvRecorder *PvRecorder) GetIsRecording() bool {
```

Comment:
I would just use this more as a property name (like we did with .NET and Python)
```suggestion
func (pvRecorder *PvRecorder) IsRecording() bool {
```